.gitmodules

```diff
@@ -10,15 +10,18 @@
 [submodule "_includes/draft-03"]
 	path = _includes/draft-03
 	url = https://github.com/json-schema-org/json-schema-spec.git
+		branch = draft-03
```

Comment:
Indentation seems wrong on these `branch` attributes

Reply:
Thanks, I'll fix that.